x-pack/plugin/security/src/test/java/org/elasticsearch/integration/IndexPrivilegeTests.java

```diff
@@ -385,6 +391,13 @@ public void testUserU14() throws Exception {
                 "GET", "/" + randomIndex() + "/_mtermvectors", "{ \"docs\" : [ { \"_id\": \"1\" }, { \"_id\": \"2\" } ] }");
     }
 
+    public void testUserU15() throws Exception {
+        assertUserIsAllowed("u15", "maintenance", "a");
+        assertUserIsDenied("u15", "crud", "a");
+
+        assertUserIsDenied("u11", "maintenance", "a");
```

Comment:
Please move this to the `testUser11` function and do it for the `b` and `c` indices, i.e.
`assertUserIsDenied("u11", "maintenance", "b");`, `assertUserIsDenied("u11", "maintenance", "c");` .